hrms/payroll/doctype/payroll_entry/payroll_entry.js

```diff
@@ -128,12 +128,7 @@ frappe.ui.form.on('Payroll Entry', {
 	create_salary_slips: function (frm) {
 		frm.call({
 			doc: frm.doc,
-			method: "run_doc_method",
-			args: {
-				method: "create_salary_slips",
-				dt: "Payroll Entry",
-				dn: frm.doc.name
-			}
+			method: "create_salary_slips"
```

Comment:
This is done for a performance boost 
https://github.com/frappe/hrms/pull/417#discussion_r1176334358

Are you facing some issue due to this?